Wazuh 4.3.7 release notes and versions update (#5489)
wazuh/wazuh-documentation: source/release-notes/release-4-3-7.rst

`#4399 <https://github.com/wazuh/wazuh-kibana-app/pull/4399>`_    The ``config-equivalences`` file folder is changed to make it available for the entire application.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    An error during the generation of a group's report, if the request to the Wazuh API fails, is fixed.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    A problem with the group's report, when the group has no agents, is fixed.
`#4352 <https://github.com/wazuh/wazuh-kibana-app/pull/4352>`_    A path in the logo customization section is fixed.
`#4362 <https://github.com/wazuh/wazuh-kibana-app/pull/4362>`_    A TypeError in a resource that fails in Chrome and Firefox browsers is fixed.
`#4358 <https://github.com/wazuh/wazuh-kibana-app/pull/4358>`_    An error of an undefined username hash related to reporting when using Kibana with X-Pack and security was disabled is fixed.
`#4359 <https://github.com/wazuh/wazuh-kibana-app/pull/4359>`_    The persistence of the plugin registry file between updates is fixed.

> `#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error on the SCA Inventory table is fixed.

Reviewer: ```suggestion
`#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error when filtering by file on the SCA Inventory is fixed.
```

`#4373 <https://github.com/wazuh/wazuh-kibana-app/pull/4373>`_    A routing loop when reinstalling the Wazuh indexer is fixed.

> `#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Wazuh restart UI is fixed.

Reviewer: ```suggestion
`#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Centralized service to restart the Wazuh managers is added.
```

`#4399 <https://github.com/wazuh/wazuh-kibana-app/pull/4399>`_    The ``config-equivalences`` file folder is changed to make it available for the entire application.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    An error during the generation of a group's report, if the request to the Wazuh API fails, is fixed.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    A problem with the group's report, when the group has no agents, is fixed.
`#4352 <https://github.com/wazuh/wazuh-kibana-app/pull/4352>`_    A path in the logo customization section is fixed.
`#4362 <https://github.com/wazuh/wazuh-kibana-app/pull/4362>`_    A TypeError in a resource that fails in Chrome and Firefox browsers is fixed.
`#4358 <https://github.com/wazuh/wazuh-kibana-app/pull/4358>`_    An error of an undefined username hash related to reporting when using Kibana with X-Pack and security was disabled is fixed.
`#4359 <https://github.com/wazuh/wazuh-kibana-app/pull/4359>`_    The persistence of the plugin registry file between updates is fixed.

> `#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error on the SCA Inventory table is fixed.

Reviewer: ```suggestion
`#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error when filtering by file on the SCA Inventory is fixed.
```

Reviewer 2: The fix is not limited to a specific field but to avoid possible errors when selecting non-existent suggested fields.

`#4373 <https://github.com/wazuh/wazuh-kibana-app/pull/4373>`_    A routing loop when reinstalling the Wazuh indexer is fixed.

> `#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Wazuh restart UI is fixed.

Reviewer: ```suggestion
`#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Centralized service to restart the Wazuh managers is added.
```

`#4399 <https://github.com/wazuh/wazuh-kibana-app/pull/4399>`_    The ``config-equivalences`` file folder is changed to make it available for the entire application.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    An error during the generation of a group's report, if the request to the Wazuh API fails, is fixed.
`#4350 <https://github.com/wazuh/wazuh-kibana-app/pull/4350>`_    A problem with the group's report, when the group has no agents, is fixed.
`#4352 <https://github.com/wazuh/wazuh-kibana-app/pull/4352>`_    A path in the logo customization section is fixed.
`#4362 <https://github.com/wazuh/wazuh-kibana-app/pull/4362>`_    A TypeError in a resource that fails in Chrome and Firefox browsers is fixed.
`#4358 <https://github.com/wazuh/wazuh-kibana-app/pull/4358>`_    An error of an undefined username hash related to reporting when using Kibana with X-Pack and security was disabled is fixed.
`#4359 <https://github.com/wazuh/wazuh-kibana-app/pull/4359>`_    The persistence of the plugin registry file between updates is fixed.

> `#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error on the SCA Inventory table is fixed.

Reviewer: ```suggestion
`#4367 <https://github.com/wazuh/wazuh-kibana-app/pull/4367>`_    A search bar error when filtering by file on the SCA Inventory is fixed.
```

`#4373 <https://github.com/wazuh/wazuh-kibana-app/pull/4373>`_    A routing loop when reinstalling the Wazuh indexer is fixed.

> `#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Wazuh restart UI is fixed.

Reviewer: ```suggestion
`#4365 <https://github.com/wazuh/wazuh-kibana-app/pull/4365>`_    Centralized service to restart the Wazuh managers is added.
```

==============================================================    =============

Wazuh Splunk app
^^^^^^^^^^^^^^^^

==============================================================    =============
Reference                                                         Description
==============================================================    =============
`#1359 <https://github.com/wazuh/wazuh-splunk/pull/1359>`_        The API console suggestions were not working in version 4.3.6 and are now fixed.
==============================================================    =============

Packages
^^^^^^^^

==============================================================    =============
Reference                                                         Description
==============================================================    =============

> `#1762 <https://github.com/wazuh/wazuh-packages/pull/1762>`__     Wazuh GPG key is now removed when uninstalling all wazuh components with the installation assistant.

Reviewer: ```suggestion
`#1762 <https://github.com/wazuh/wazuh-packages/pull/1762>`__     The Wazuh GPG key not being removed when uninstalling all the Wazuh components using the installation assistant is fixed.
```

> `#1765 <https://github.com/wazuh/wazuh-packages/pull/1765>`__     Some errors related to filebeat downloaded files in filebeat installation are fixed.

Reviewer: ```suggestion
`#1765 <https://github.com/wazuh/wazuh-packages/pull/1765>`__     Handling of errors that might happen when downloading Filebeat files is added.
```

> `#1766 <https://github.com/wazuh/wazuh-packages/pull/1766>`__     An indentation issue in ``config.yml`` is fixed because certificates were not created correctly if they have a bad indentation.

Reviewer: ```suggestion
`#1766 <https://github.com/wazuh/wazuh-packages/pull/1766>`__     A check of indentations in the ``config.yml`` file is added.
```

> `#1731 <https://github.com/wazuh/wazuh-packages/pull/1731>`_      IP error in the distributed configuration when installing every component in the same host is fixed.

Reviewer: ```suggestion
`#1731 <https://github.com/wazuh/wazuh-packages/pull/1731>`_      An error when installing every component of a distributed installation in the same host using the 127.0.0.1 IP address is fixed.
```

> `#1619 <https://github.com/wazuh/wazuh-packages/pull/1619>`_      All warnings on the Wazuh installation assistant when running ``shellcheck`` manually are fixed.

Reviewer: ```suggestion
`#1619 <https://github.com/wazuh/wazuh-packages/pull/1619>`_      The code of the Wazuh installation assistant has been improved.
```
